src/web/components/Contributors/Contributors.tsx

```diff
@@ -0,0 +1,66 @@
+        { name: "Sacha Narinx", avatar: "https://avatars.githubusercontent.com/u/2101287?v=4?s=100", roles: ["💻", "👀", "📖"], profileLink: "https://github.com/Springstone" },
+        { name: "Jamel Achahbar", avatar: "https://avatars.githubusercontent.com/u/127963872?v=4?s=100", roles: ["💻"], profileLink: "https://github.com/jamelachahbar" },
+        { name: "Saad Mahmood", avatar: "https://avatars.githubusercontent.com/u/66096650?v=4?s=100", roles: ["💻"], profileLink: "https://github.com/saadmsft" },
+        { name: "Divyadeep Dayal", avatar: "https://avatars.githubusercontent.com/u/81250915?v=4?s=100", roles: ["💻"], profileLink: "https://github.com/didayal-msft" },
+        { name: "Arjen Huitema", avatar: "https://avatars.githubusercontent.com/u/15944031?v=4?s=100", roles: ["💻"], profileLink: "https://github.com/arjenhuitema" },
+        { name: "Bill Anderson", avatar: "https://avatars.githubusercontent.com/u/9596428?v=4?s=100", roles: ["📖"], profileLink: "https://github.com/bandersmsft" },
+        { name: "Hélder Pinto", avatar: "https://avatars.githubusercontent.com/u/10661605?v=4?s=100", roles: ["💻", "👀", "📖", "🐛"], profileLink: "https://aka.ms/helderpinto" },
+        { name: "Yuan Zhang", avatar: "https://avatars.githubusercontent.com/u/114724932?v=4?s=100", roles: ["💻"], profileLink: "https://aka.ms/yuanzhang9" },
+        { name: "ymehdimsft", avatar: "https://avatars.githubusercontent.com/u/134303029?v=4?s=100", roles: ["💻"], profileLink: "https://github.com/ymehdimsft" },
+        { name: "srilatha inavolu", avatar: "https://avatars.githubusercontent.com/u/4493254?v=4?s=100", roles: ["💻", "👀"], profileLink: "https://github.com/sri-" },
+        { name: "soumyananda", avatar: "https://avatars.githubusercontent.com/u/7952916?v=4?s=100", roles: ["💻", "👀"], profileLink: "https://github.com/soumyananda" },
+        { name: "Chris Bowman", avatar: "https://avatars.githubusercontent.com/u/20289947?v=4?s=100", roles: ["🐛"], profileLink: "https://github.com/chris-bowman" },
+        { name: "Mubarak Tanseer", avatar: "https://avatars.githubusercontent.com/u/64589176?v=4?s=100", roles: ["🐛"], profileLink: "https://github.com/mutansee" },
+        { name: "Ben Shy", avatar: "https://avatars.githubusercontent.com/u/18198475?v=4?s=100", roles: ["💻", "👀"], profileLink: "https://github.com/BenShy" },
+        { name: "Trey Morgan", avatar: "https://avatars.githubusercontent.com/u/18508457?v=4?s=100", roles: ["💻"], profileLink: "https://github.com/treymorgan" },
+    ];
+
+    return (
+        <>
+            <div className="ftk-contributors-title">
+                <Text size={600} weight="semibold">Contributors</Text>
+            </div>
+
+            <div className="ftk-background-card">
+                <div className="ftk-left-section">
+                    <Text size={500} weight="regular">Get help from the community or contribute to the project</Text>
```

Comment:
Do you want feedback on aligning to the content in prod today or do you want to do that in a separate PR? I'm fine either way.

https://aka.ms/finops/toolkit